elide-standalone/README.md

```diff
@@ -36,23 +37,298 @@ To include `elide-standalone` into your project, add the single dependency:
+
+### App & Settings
+
+Bringing life to our API is trivially easy. We need two new classes: Main and Settings.
+
+#### Main.java
+
+  ```java
+  public class Main {
+      public static void main(String[] args) throws Exception {
+          ElideStandalone app = new ElideStandalone(new Settings());
+          app.start();
+      }
+  }
+  ```
+
+#### Settings.java
+
+  ```java
+  public class Settings implements ElideStandaloneSettings {
+      @Override
+      public String getModelPackageName() {
+          return ArtifactGroup.class.getPackage().getName();
+      }
+
+      @Override
+      public Properties getDatabaseProperties() {
+          Properties options = new Properties();
+          
+          //Here we use H2 in memory instead of Postgres 
+          options.put("hibernate.dialect", "org.hibernate.dialect.H2Dialect");
+          options.put("javax.persistence.jdbc.driver", "org.h2.Driver");
+          options.put("javax.persistence.jdbc.url", "jdbc:h2:mem:db1;DB_CLOSE_DELAY=-1");
+          options.put("javax.persistence.jdbc.user", "sa");
+          options.put("javax.persistence.jdbc.password", "");
+             
+          return options;
+      }
+  } 
+  ```
+
+### Supporting Files
+
+Elide standalone uses a JPA data store (the thing that talks to the database) that is [configured programmatically](https://github.com/aklish/elide-heroku-example/blob/master/src/main/java/example/Settings.java#L95-L111) (no persistence.xml required).
+
+If you want to see the logs from your shiny new API, you will also want a [log4j config](https://github.com/aklish/elide-heroku-example/blob/master/src/main/resources/log4j2.xml).
+Your log4j config should go in `src/main/resources` so log4j can find it.
+
+### Running
+
+With these new classes, you have two options for running your project.  You can either run the `Main` class using your
+favorite IDE, or you can run the service from the command line:
+
+```mvn exec:java -Dexec.mainClass="example.Main"```
+
+Our example requires the following environment variables to be set to work correctly with Heroku and Postgres.  
+
+1. JDBC_DATABASE_URL
+2. JDBC_DATABASE_USERNAME
+3. JDBC_DATABASE_PASSWORD
+
+If running inside a Heroku dyno, Heroku sets these variables for us.  If you don't set them, the example will use the H2 in memory database.
+
+With the `Main` and `Settings` classes we can now run our API. 
+
+You can now run the following curl commands to see some of the sample data that the liquibase migrations added for us:
+Don't forget to replace localhost:8080 with your Heroku URL if running from Heroku!
+
+#### JSON-API
+
+  ```curl
+  curl http://localhost:8080/api/v1/group
+  ```
+
+#### GraphQL
+  ```curl
+  curl -g -X POST -H"Content-Type: application/json" -H"Accept: application/json" \
+      "http://localhost:8080/graphql/api/v1" \
+      -d'{   
+             "query" : "{ group { edges { node { name commonName description } } } }"
```

Comment:
The graphql request example in elide.io only contains this part `{ group { edges { node { name commonName description } } } }`, we should also update that one too. And I think we can also explain what this query means.